lib/rubill/invoice.rb

```diff
@@ -28,10 +28,10 @@ def amount_due
       remote_record[:amountDue]
     end
 
-    def self.line_item(amount, description, item_id)
+    def self.line_item(amount, description, item_id, quantity = 1)
```

Comment:
also if I were a hound I would say

Do not use spaces around default parameter assignment at 31,70